pages/docs/drop-in-components/index.mdx

```diff
@@ -994,9 +994,18 @@ Get started building with drop-in components by checking out the [getting starte
 
 ## Changelog
 
-### Latest `v2.2.1`
+### `v2.2.2` (Latest)
 
-The latest version of **dwolla-web.js** is `2.2.1`. If you are currently using `v2`, we recommend [upgrading](https://developers.dwolla.com/docs/drop-in-components/drop-ins/setup-and-configuration#include-dwolla-webjs) to `2.2.1`.
+The latest version of **dwolla-web.js** is `2.2.2`. If you are currently using `v2`, we recommend [upgrading](https://developers.dwolla.com/docs/drop-in-components/drop-ins/setup-and-configuration#include-dwolla-webjs) to `2.2.2`.
+
+- Updated user hints and prompts for documentation upload screens to be correct and consistent throughout the following drop-in components:
+  - `<dwolla-account-opening>`
```

Comment:
@ShreyaThapa Is this a new component?

Reply:
Not a new component. Just realized we don't document this component. I think I'll just remove it's mention.

Reply:
We do mention it in the changelog though - https://developers.dwolla.com/docs/drop-in-components#v219

Open to ideas on whether we should continue documenting in the changelog or not.